Jenkinsfile_CNP

```diff
@@ -30,8 +36,13 @@ def vaultOverrides = [
     'spreview': 'saat'
 ]
 // Vars needed for AKS functional and smoke tests
+env.IDAM_URL = "https://idam-api.aat.platform.hmcts.net"
 env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
 env.CCD_DS_SERVICE_NAME = "ccd_data"
+env.CCD_GW_SERVICE_NAME = "ccd_gw"
```

Comment:
These 2 can be replaced by only: BEFTA_S2S_CLIENT_ID.
The value will be 'ccd_data'.